Implement Server-Side sessions (#68117)
elastic/kibana: docs/user/security/securing-kibana.asciidoc

For more information, see <<security-settings-kb,Security settings in {kib}>>.
--

> . Configure {kib}'s session expiration settings. We strongly recommend setting both idle timeout and lifespan settings:

Reviewer: ```suggestion
. Configure {kib}'s session expiration settings. Set both the idle timeout and lifespan settings:
```
